internal/pkg/cli/app_delete.go

```diff
@@ -166,13 +187,19 @@ func (o *deleteAppOpts) retrieveAppNames() ([]string, error) {
 }
 
 func (o *deleteAppOpts) sourceProjectEnvironments() error {
-	envs, err := o.projectService.ListEnvironments(o.ProjectName())
 
-	if err != nil {
-		return fmt.Errorf("get environments: %w", err)
-	}
+	if o.EnvName != "" {
+		env, _ := o.targetEnv()
```

Comment:
I think we might not want to ignore the error handling here because if anything happen (like parameter gets deleted) after the validation part, then it increases the difficulty to find the bug.